Utilities/build-script-helper.py

```diff
@@ -105,7 +108,10 @@ def get_swiftpm_options(swift_exec: str, args: argparse.Namespace) -> List[str]:
         for san in args.sanitize:
             swiftpm_args += ['--sanitize=%s' % san]
 
-    if platform.system() == 'Darwin':
+    build_target = get_build_target(swift_exec, args, cross_compile=(True if args.cross_compile_config else False))
+    build_arch = build_target.split('-')[0]
+    build_os = build_target.split('-')[2]
+    if re.search('-apple-macosx', build_target):
```

Comment:
Again, can’t we just use substring matching here?

```suggestion
    if '-apple-macosx' in build_target:
```

Alternatively, wouldn’t it even be better to just check the OS? I.e something like `build_os.startswith('macosx')`

Reply:
Will do.